### DISCUSSION TITLE
computing seal proof failed(2): storage call error 0: Rust panic: no unwind information

### DISCUSSION THREAD

**Discussion by ChaiWenyu1983** [filecoin-project/lotus#9214] [Storage Provider] [answered]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [ ] I did not make any code changes to lotus.

### Lotus component

- [ ] lotus daemon - chain sync
- [ ] lotus miner - mining and block production
- [X] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt)
- [ ] lotus miner/market - storage deal
- [ ] lotus miner/market - retrieval deal
- [ ] lotus miner/market - data transfer
- [ ] lotus client
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [ ] Other

### Lotus Version

```text
Daemon:  1.17.0+mainnet+git.2830429ad.dirty+api1.5.0
Local: lotus version 1.17.0+mainnet+git.2830429ad.dirty
Local: lotus-miner version 1.17.0+mainnet+git.2830429ad.dirty
```

### Describe the Bug

There are three tasks, which are in the "C2" stage, and suddenly fail to work.
The logs are as follows:
SectorID:	594
Status:		PreCommit1
CIDcommD:	baga6ea4seaqao7s73y24kcutaosvacpdjgfe5pw76ooefnyqw4ynr3d2y6x2mpq
CIDcommR:	bagboea4b5abcb7fwt2z2apdfaozrkve5gynrmuceeljp6y6lrfdqye3g46zkpssn
Ticket:		1039a2a167c53d0c480bb51c008cadb477d6a0ed5ed9b34afd20c882c254fda0
TicketH:	2099889
Seed:		d2d00adef8af2186d0bf700790a87f779a1b33b8dad1dc173022037ce19bd2c2
SeedH:		2101436
Precommit:	bafy2bzaceaezldzboxldq5tsnivxtr55p7advvvr53kvyow2ksbwdwkz2ox5i
Commit:		<nil>
Deals:		[0]
Retries:	0
--------
Event Log:
0.	2022-08-24 16:32:01 +0800 CST:	[event;sealing.SectorStartCC]	{"User":{"ID":594,"SectorType":8}}
1.	2022-08-24 16:34:47 +0800 CST:	[event;sealing.SectorPacked]	{"User":{"FillerPieces":[{"Size":34359738368,"PieceCID":{"/":"baga6ea4seaqao7s73y24kcutaosvacpdjgfe5pw76ooefnyqw4ynr3d2y6x2mpq"}}]}}
2.	2022-08-24 16:34:47 +0800 CST:	[event;sealing.SectorTicket]	{"User":{"TicketValue":"EDmioWfFPQxIC7UcAIyttHfWoO1e2bNK/SDIgsJU/aA=","TicketEpoch":2099889}}
3.	2022-08-24 16:34:47 +0800 CST:	[event;sealing.SectorSealPreCommit1Failed]	{"User":{}}
	seal pre commit(1) failed: storage call error 102: can't reserve 484472310988 bytes in '/mnt/ssd3/sealing' (id:cdf9141e-9aaf-49b8-b3a0-ea7b94c52b69), only 137965635176 available
4.	2022-08-24 16:35:47 +0800 CST:	[event;sealing.SectorRetrySealPreCommit1]	{"User":{}}
5.	2022-08-24 20:07:46 +0800 CST:	[event;sealing.SectorPreCommit1]	{"User":{"PreCommit1Out":"eyJfbG90dXNfU2VhbFJhbmRvbW5lc3MiOiJFRG1pb1dmRlBReElDN1VjQUl5dHRIZldvTzFlMmJOSy9TRElnc0pVL2FBPSIsImNvbW1fZCI6WzcsMTI2LDk1LDIyMiw1MywxOTcsMTAsMTQ3LDMsMTY1LDgwLDksMjI3LDczLDEzOCw3OCwxOTAsMjIzLDI0MywxNTYsNjYsMTgzLDE2LDE4Myw0OCwyMTYsMjM2LDEyMiwxOTksMTc1LDE2Niw2Ml0sImNvbmZpZyI6eyJpZCI6InRyZWUtZCIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC01OTQiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjIxNDc0ODM2NDd9LCJsYWJlbHMiOnsiU3RhY2tlZERyZzMyR2lCVjEiOnsiX2giOm51bGwsImxhYmVscyI6W3siaWQiOiJsYXllci0xIiwicGF0aCI6Ii9tbnQvc3NkMy9zZWFsaW5nL2NhY2hlL3MtdDAxODYyNTY0LTU5NCIsInJvd3NfdG9fZGlzY2FyZCI6Nywic2l6ZSI6MTA3Mzc0MTgyNH0seyJpZCI6ImxheWVyLTIiLCJwYXRoIjoiL21udC9zc2QzL3NlYWxpbmcvY2FjaGUvcy10MDE4NjI1NjQtNTk0Iiwicm93c190b19kaXNjYXJkIjo3LCJzaXplIjoxMDczNzQxODI0fSx7ImlkIjoibGF5ZXItMyIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC01OTQiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjEwNzM3NDE4MjR9LHsiaWQiOiJsYXllci00IiwicGF0aCI6Ii9tbnQvc3NkMy9zZWFsaW5nL2NhY2hlL3MtdDAxODYyNTY0LTU5NCIsInJvd3NfdG9fZGlzY2FyZCI6Nywic2l6ZSI6MTA3Mzc0MTgyNH0seyJpZCI6ImxheWVyLTUiLCJwYXRoIjoiL21udC9zc2QzL3NlYWxpbmcvY2FjaGUvcy10MDE4NjI1NjQtNTk0Iiwicm93c190b19kaXNjYXJkIjo3LCJzaXplIjoxMDczNzQxODI0fSx7ImlkIjoibGF5ZXItNiIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC01OTQiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjEwNzM3NDE4MjR9LHsiaWQiOiJsYXllci03IiwicGF0aCI6Ii9tbnQvc3NkMy9zZWFsaW5nL2NhY2hlL3MtdDAxODYyNTY0LTU5NCIsInJvd3NfdG9fZGlzY2FyZCI6Nywic2l6ZSI6MTA3Mzc0MTgyNH0seyJpZCI6ImxheWVyLTgiLCJwYXRoIjoiL21udC9zc2QzL3NlYWxpbmcvY2FjaGUvcy10MDE4NjI1NjQtNTk0Iiwicm93c190b19kaXNjYXJkIjo3LCJzaXplIjoxMDczNzQxODI0fSx7ImlkIjoibGF5ZXItOSIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC01OTQiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjEwNzM3NDE4MjR9LHsiaWQiOiJsYXllci0xMCIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC01OTQiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjEwNzM3NDE4MjR9LHsiaWQiOiJsYXllci0xMSIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC01OTQiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjEwNzM3NDE4MjR9XX19LCJyZWdpc3RlcmVkX3Byb29mIjoiU3RhY2tlZERyZzMyR2lCVjFfMSJ9"}}
6.	2022-08-24 20:41:39 +0800 CST:	[event;sealing.SectorPreCommit2]	{"User":{"Sealed":{"/":"bagboea4b5abcb7fwt2z2apdfaozrkve5gynrmuceeljp6y6lrfdqye3g46zkpssn"},"Unsealed":{"/":"baga6ea4seaqao7s73y24kcutaosvacpdjgfe5pw76ooefnyqw4ynr3d2y6x2mpq"}}}
7.	2022-08-24 20:41:39 +0800 CST:	[event;sealing.SectorPreCommitBatch]{"User":{}}
8.	2022-08-24 20:42:01 +0800 CST:	[event;sealing.SectorPreCommitBatchSent]	{"User":{"Message":{"/":"bafy2bzaceaezldzboxldq5tsnivxtr55p7advvvr53kvyow2ksbwdwkz2ox5i"}}}
9.	2022-08-24 20:46:00 +0800 CST:	[event;sealing.SectorPreCommitLanded]	{"User":{"TipSet":[{"/":"bafy2bzacedcbzh255xdvswhcsi2iywam5gowaobss6vii3i74ily7igx2r3ui"},{"/":"bafy2bzacedfgsi4ecv2nauwqghp5icp2usv6wfll2u6f6l2xwqa4qcuisktge"},{"/":"bafy2bzacedaz3zvlyjmbrme2iva4x36ixndtzrj4zyoigbktip3dafqhw7fkm"},{"/":"bafy2bzaceclings6zhnynodz2zd3yeeng7ytul5bzlggen3hws4irjidwt7pe"},{"/":"bafy2bzaceaf5jkh2hineinb62mjp35fpb5rhrllm7mhitrlgbs7g22xgull4k"},{"/":"bafy2bzacearhvtz72udnww5xdtasgkzcamr64sugikv6x2smpkhcniyravuto"},{"/":"bafy2bzacea46knkj5yx3khoytv5eholvmmjbbw53iw24rcitrulqt53xbyk6o"},{"/":"bafy2bzacebzgxifseaftf2jyxhtgxufe6myvmabe5gzls7cxonl4m6k6lne7g"},{"/":"bafy2bzaceaaafmufga73ege74ga6esa7savaf6h2i3amwdilxbkoayyjmh3la"}]}}
10.	2022-08-24 22:01:03 +0800 CST:	[event;sealing.SectorSeedReady]	{"User":{"SeedValue":"0tAK3vivIYbQv3AHkKh/d5obM7ja0dwXMCIDfOGb0sI=","SeedEpoch":2101436}}
11.	2022-08-24 22:20:07 +0800 CST:	[event;sealing.SectorComputeProofFailed]	{"User":{}}
	computing seal proof failed(2): storage call error 0: Rust panic: no unwind information [Hostname: chaiwenyu-System-Product-Name]: Rust panic: no unwind information
12.	2022-08-24 22:21:07 +0800 CST:	[event;sealing.SectorRetryComputeProof]	{"User":{}}
13.	2022-08-24 22:25:22 +0800 CST:	[event;sealing.SectorComputeProofFailed]	{"User":{}}
	computing seal proof failed(2): storage call error 0: Rust panic: no unwind information [Hostname: chaiwenyu-System-Product-Name]: Rust panic: no unwind information
14.	2022-08-24 22:26:22 +0800 CST:	[event;sealing.SectorRetryComputeProof]	{"User":{}}
15.	2022-08-24 22:30:28 +0800 CST:	[event;sealing.SectorComputeProofFailed]	{"User":{}}
	computing seal proof failed(2): storage call error 0: Rust panic: no unwind information [Hostname: chaiwenyu-System-Product-Name]: Rust panic: no unwind information
16.	2022-08-24 22:31:28 +0800 CST:	[event;sealing.SectorSealPreCommit1Failed]	{"User":{}}
	consecutive compute fails
17.	2022-08-24 22:32:28 +0800 CST:	[event;sealing.SectorRetrySealPreCommit1]	{"User":{}}

SectorID:	595
Status:		PreCommit1
CIDcommD:	baga6ea4seaqao7s73y24kcutaosvacpdjgfe5pw76ooefnyqw4ynr3d2y6x2mpq
CIDcommR:	bagboea4b5abcadxgp4gd72ofs4rignkcsz5mvfvp72milyzvm7pzk2nxut3f45sf
Ticket:		1039a2a167c53d0c480bb51c008cadb477d6a0ed5ed9b34afd20c882c254fda0
TicketH:	2099889
Seed:		cc88c4d6c835fd7ad91ee042d8a8e78bcf3938c2c77ffa6383c09133b760668e
SeedH:		2101387
Precommit:	bafy2bzaceclga4ga6nwtwdcfaayeqf42cml4po333hurmoeaitqzwbx5bl75u
Commit:		<nil>
Deals:		[0]
Retries:	0
--------
Event Log:
0.	2022-08-24 16:32:01 +0800 CST:	[event;sealing.SectorStartCC]	{"User":{"ID":595,"SectorType":8}}
1.	2022-08-24 16:34:47 +0800 CST:	[event;sealing.SectorPacked]	{"User":{"FillerPieces":[{"Size":34359738368,"PieceCID":{"/":"baga6ea4seaqao7s73y24kcutaosvacpdjgfe5pw76ooefnyqw4ynr3d2y6x2mpq"}}]}}
2.	2022-08-24 16:34:47 +0800 CST:	[event;sealing.SectorTicket]	{"User":{"TicketValue":"EDmioWfFPQxIC7UcAIyttHfWoO1e2bNK/SDIgsJU/aA=","TicketEpoch":2099889}}
3.	2022-08-24 20:04:33 +0800 CST:	[event;sealing.SectorPreCommit1]	{"User":{"PreCommit1Out":"eyJfbG90dXNfU2VhbFJhbmRvbW5lc3MiOiJFRG1pb1dmRlBReElDN1VjQUl5dHRIZldvTzFlMmJOSy9TRElnc0pVL2FBPSIsImNvbW1fZCI6WzcsMTI2LDk1LDIyMiw1MywxOTcsMTAsMTQ3LDMsMTY1LDgwLDksMjI3LDczLDEzOCw3OCwxOTAsMjIzLDI0MywxNTYsNjYsMTgzLDE2LDE4Myw0OCwyMTYsMjM2LDEyMiwxOTksMTc1LDE2Niw2Ml0sImNvbmZpZyI6eyJpZCI6InRyZWUtZCIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC01OTUiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjIxNDc0ODM2NDd9LCJsYWJlbHMiOnsiU3RhY2tlZERyZzMyR2lCVjEiOnsiX2giOm51bGwsImxhYmVscyI6W3siaWQiOiJsYXllci0xIiwicGF0aCI6Ii9tbnQvc3NkMy9zZWFsaW5nL2NhY2hlL3MtdDAxODYyNTY0LTU5NSIsInJvd3NfdG9fZGlzY2FyZCI6Nywic2l6ZSI6MTA3Mzc0MTgyNH0seyJpZCI6ImxheWVyLTIiLCJwYXRoIjoiL21udC9zc2QzL3NlYWxpbmcvY2FjaGUvcy10MDE4NjI1NjQtNTk1Iiwicm93c190b19kaXNjYXJkIjo3LCJzaXplIjoxMDczNzQxODI0fSx7ImlkIjoibGF5ZXItMyIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC01OTUiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjEwNzM3NDE4MjR9LHsiaWQiOiJsYXllci00IiwicGF0aCI6Ii9tbnQvc3NkMy9zZWFsaW5nL2NhY2hlL3MtdDAxODYyNTY0LTU5NSIsInJvd3NfdG9fZGlzY2FyZCI6Nywic2l6ZSI6MTA3Mzc0MTgyNH0seyJpZCI6ImxheWVyLTUiLCJwYXRoIjoiL21udC9zc2QzL3NlYWxpbmcvY2FjaGUvcy10MDE4NjI1NjQtNTk1Iiwicm93c190b19kaXNjYXJkIjo3LCJzaXplIjoxMDczNzQxODI0fSx7ImlkIjoibGF5ZXItNiIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC01OTUiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjEwNzM3NDE4MjR9LHsiaWQiOiJsYXllci03IiwicGF0aCI6Ii9tbnQvc3NkMy9zZWFsaW5nL2NhY2hlL3MtdDAxODYyNTY0LTU5NSIsInJvd3NfdG9fZGlzY2FyZCI6Nywic2l6ZSI6MTA3Mzc0MTgyNH0seyJpZCI6ImxheWVyLTgiLCJwYXRoIjoiL21udC9zc2QzL3NlYWxpbmcvY2FjaGUvcy10MDE4NjI1NjQtNTk1Iiwicm93c190b19kaXNjYXJkIjo3LCJzaXplIjoxMDczNzQxODI0fSx7ImlkIjoibGF5ZXItOSIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC01OTUiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjEwNzM3NDE4MjR9LHsiaWQiOiJsYXllci0xMCIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC01OTUiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjEwNzM3NDE4MjR9LHsiaWQiOiJsYXllci0xMSIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC01OTUiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjEwNzM3NDE4MjR9XX19LCJyZWdpc3RlcmVkX3Byb29mIjoiU3RhY2tlZERyZzMyR2lCVjFfMSJ9"}}
4.	2022-08-24 20:16:38 +0800 CST:	[event;sealing.SectorPreCommit2]	{"User":{"Sealed":{"/":"bagboea4b5abcadxgp4gd72ofs4rignkcsz5mvfvp72milyzvm7pzk2nxut3f45sf"},"Unsealed":{"/":"baga6ea4seaqao7s73y24kcutaosvacpdjgfe5pw76ooefnyqw4ynr3d2y6x2mpq"}}}
5.	2022-08-24 20:16:38 +0800 CST:	[event;sealing.SectorPreCommitBatch]{"User":{}}
6.	2022-08-24 20:18:02 +0800 CST:	[event;sealing.SectorPreCommitBatchSent]	{"User":{"Message":{"/":"bafy2bzaceclga4ga6nwtwdcfaayeqf42cml4po333hurmoeaitqzwbx5bl75u"}}}
7.	2022-08-24 20:21:29 +0800 CST:	[event;sealing.SectorPreCommitLanded]	{"User":{"TipSet":[{"/":"bafy2bzaceatfwd42bbevna4uejuxyiregv56zziwwe3lfjktdgk3k6jsrm5ee"},{"/":"bafy2bzacedmsxn2eg2ef7ssa2225cowvqriojrcqgghrz3qmg5kyepsii7zcw"},{"/":"bafy2bzacedf6dwglyptowyrenbqgrehhnmbvtzh4byvzjx4x2u4s2audbvvna"},{"/":"bafy2bzaceaszin6kqbmzfpdheexcfk23n4ti5fkweftyx74oqdq4ipbgxvkq6"}]}}
8.	2022-08-24 21:36:30 +0800 CST:	[event;sealing.SectorSeedReady]	{"User":{"SeedValue":"zIjE1sg1/XrZHuBC2Kjni885OMLHf/pjg8CRM7dgZo4=","SeedEpoch":2101387}}
9.	2022-08-24 22:16:22 +0800 CST:	[event;sealing.SectorComputeProofFailed]	{"User":{}}
	computing seal proof failed(2): storage call error 0: Rust panic: Worker Pool was poisoned [Hostname: chaiwenyu-System-Product-Name]: Rust panic: Worker Pool was poisoned
10.	2022-08-24 22:17:22 +0800 CST:	[event;sealing.SectorRetryComputeProof]	{"User":{}}
11.	2022-08-24 22:21:53 +0800 CST:	[event;sealing.SectorComputeProofFailed]	{"User":{}}
	computing seal proof failed(2): storage call error 0: Rust panic: no unwind information [Hostname: chaiwenyu-System-Product-Name]: Rust panic: no unwind information
12.	2022-08-24 22:22:53 +0800 CST:	[event;sealing.SectorRetryComputeProof]	{"User":{}}
13.	2022-08-24 22:27:04 +0800 CST:	[event;sealing.SectorComputeProofFailed]	{"User":{}}
	computing seal proof failed(2): storage call error 0: Rust panic: no unwind information [Hostname: chaiwenyu-System-Product-Name]: Rust panic: no unwind information
14.	2022-08-24 22:28:04 +0800 CST:	[event;sealing.SectorSealPreCommit1Failed]	{"User":{}}
	consecutive compute fails
15.	2022-08-24 22:29:04 +0800 CST:	[event;sealing.SectorRetrySealPreCommit1]	{"User":{}}

SectorID:	596
Status:		PreCommit1
CIDcommD:	baga6ea4seaqao7s73y24kcutaosvacpdjgfe5pw76ooefnyqw4ynr3d2y6x2mpq
CIDcommR:	bagboea4b5abcakt5lunssux3eprppjorpmzbxj2sfcp2db7jqcccwvqehm4l44ry
Ticket:		1039a2a167c53d0c480bb51c008cadb477d6a0ed5ed9b34afd20c882c254fda0
TicketH:	2099889
Seed:		9e14b9aef3363e6f8962e2bf7e9bba7cff4b1f6b982e59f504d03622995fd376
SeedH:		2101411
Precommit:	bafy2bzacedwps2vhofko3uo7tjcuv27ud5znkxejr42mvg2a2gq3iobeys3i2
Commit:		<nil>
Deals:		[0]
Retries:	0
--------
Event Log:
0.	2022-08-24 16:32:01 +0800 CST:	[event;sealing.SectorStartCC]	{"User":{"ID":596,"SectorType":8}}
1.	2022-08-24 16:34:47 +0800 CST:	[event;sealing.SectorPacked]	{"User":{"FillerPieces":[{"Size":34359738368,"PieceCID":{"/":"baga6ea4seaqao7s73y24kcutaosvacpdjgfe5pw76ooefnyqw4ynr3d2y6x2mpq"}}]}}
2.	2022-08-24 16:34:47 +0800 CST:	[event;sealing.SectorTicket]	{"User":{"TicketValue":"EDmioWfFPQxIC7UcAIyttHfWoO1e2bNK/SDIgsJU/aA=","TicketEpoch":2099889}}
3.	2022-08-24 20:04:33 +0800 CST:	[event;sealing.SectorPreCommit1]	{"User":{"PreCommit1Out":"eyJfbG90dXNfU2VhbFJhbmRvbW5lc3MiOiJFRG1pb1dmRlBReElDN1VjQUl5dHRIZldvTzFlMmJOSy9TRElnc0pVL2FBPSIsImNvbW1fZCI6WzcsMTI2LDk1LDIyMiw1MywxOTcsMTAsMTQ3LDMsMTY1LDgwLDksMjI3LDczLDEzOCw3OCwxOTAsMjIzLDI0MywxNTYsNjYsMTgzLDE2LDE4Myw0OCwyMTYsMjM2LDEyMiwxOTksMTc1LDE2Niw2Ml0sImNvbmZpZyI6eyJpZCI6InRyZWUtZCIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC01OTYiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjIxNDc0ODM2NDd9LCJsYWJlbHMiOnsiU3RhY2tlZERyZzMyR2lCVjEiOnsiX2giOm51bGwsImxhYmVscyI6W3siaWQiOiJsYXllci0xIiwicGF0aCI6Ii9tbnQvc3NkMy9zZWFsaW5nL2NhY2hlL3MtdDAxODYyNTY0LTU5NiIsInJvd3NfdG9fZGlzY2FyZCI6Nywic2l6ZSI6MTA3Mzc0MTgyNH0seyJpZCI6ImxheWVyLTIiLCJwYXRoIjoiL21udC9zc2QzL3NlYWxpbmcvY2FjaGUvcy10MDE4NjI1NjQtNTk2Iiwicm93c190b19kaXNjYXJkIjo3LCJzaXplIjoxMDczNzQxODI0fSx7ImlkIjoibGF5ZXItMyIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC01OTYiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjEwNzM3NDE4MjR9LHsiaWQiOiJsYXllci00IiwicGF0aCI6Ii9tbnQvc3NkMy9zZWFsaW5nL2NhY2hlL3MtdDAxODYyNTY0LTU5NiIsInJvd3NfdG9fZGlzY2FyZCI6Nywic2l6ZSI6MTA3Mzc0MTgyNH0seyJpZCI6ImxheWVyLTUiLCJwYXRoIjoiL21udC9zc2QzL3NlYWxpbmcvY2FjaGUvcy10MDE4NjI1NjQtNTk2Iiwicm93c190b19kaXNjYXJkIjo3LCJzaXplIjoxMDczNzQxODI0fSx7ImlkIjoibGF5ZXItNiIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC01OTYiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjEwNzM3NDE4MjR9LHsiaWQiOiJsYXllci03IiwicGF0aCI6Ii9tbnQvc3NkMy9zZWFsaW5nL2NhY2hlL3MtdDAxODYyNTY0LTU5NiIsInJvd3NfdG9fZGlzY2FyZCI6Nywic2l6ZSI6MTA3Mzc0MTgyNH0seyJpZCI6ImxheWVyLTgiLCJwYXRoIjoiL21udC9zc2QzL3NlYWxpbmcvY2FjaGUvcy10MDE4NjI1NjQtNTk2Iiwicm93c190b19kaXNjYXJkIjo3LCJzaXplIjoxMDczNzQxODI0fSx7ImlkIjoibGF5ZXItOSIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC01OTYiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjEwNzM3NDE4MjR9LHsiaWQiOiJsYXllci0xMCIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC01OTYiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjEwNzM3NDE4MjR9LHsiaWQiOiJsYXllci0xMSIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC01OTYiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjEwNzM3NDE4MjR9XX19LCJyZWdpc3RlcmVkX3Byb29mIjoiU3RhY2tlZERyZzMyR2lCVjFfMSJ9"}}
4.	2022-08-24 20:29:06 +0800 CST:	[event;sealing.SectorPreCommit2]	{"User":{"Sealed":{"/":"bagboea4b5abcakt5lunssux3eprppjorpmzbxj2sfcp2db7jqcccwvqehm4l44ry"},"Unsealed":{"/":"baga6ea4seaqao7s73y24kcutaosvacpdjgfe5pw76ooefnyqw4ynr3d2y6x2mpq"}}}
5.	2022-08-24 20:29:06 +0800 CST:	[event;sealing.SectorPreCommitBatch]{"User":{}}
6.	2022-08-24 20:30:02 +0800 CST:	[event;sealing.SectorPreCommitBatchSent]	{"User":{"Message":{"/":"bafy2bzacedwps2vhofko3uo7tjcuv27ud5znkxejr42mvg2a2gq3iobeys3i2"}}}
7.	2022-08-24 20:33:30 +0800 CST:	[event;sealing.SectorPreCommitLanded]	{"User":{"TipSet":[{"/":"bafy2bzacebmwylzlofsu2pm6pqbt4sonuruk27lvwd52wndnsym2ctqh5ro4s"},{"/":"bafy2bzacedegnyclqcx5y5x4csdafirb6jua2ji7gro53igwzj4nazxpvyqws"},{"/":"bafy2bzacecnuj4ttaflkpchskxbtecr2qvoyboaewaqup7qbpw4ncyuacurws"},{"/":"bafy2bzaceanrbgcigy33nnbjcdstkyc2tslsmakreimby22yhfung7sr2imnq"},{"/":"bafy2bzacecrsz63o2fzzxy2xvn54watsrcwjrwzxi22svn2f5fvyt7vlxhv7a"},{"/":"bafy2bzaceclmo54dedfwvecfwnxz352nybdcdjlqz3mqysgx7iio43xpiwh4k"}]}}
8.	2022-08-24 21:48:30 +0800 CST:	[event;sealing.SectorSeedReady]	{"User":{"SeedValue":"nhS5rvM2Pm+JYuK/fpu6fP9LH2uYLln1BNA2Iplf03Y=","SeedEpoch":2101411}}
9.	2022-08-24 22:18:23 +0800 CST:	[event;sealing.SectorComputeProofFailed]	{"User":{}}
	computing seal proof failed(2): storage call error 0: Rust panic: no unwind information [Hostname: chaiwenyu-System-Product-Name]: Rust panic: no unwind information
10.	2022-08-24 22:19:23 +0800 CST:	[event;sealing.SectorRetryComputeProof]	{"User":{}}
11.	2022-08-24 22:23:37 +0800 CST:	[event;sealing.SectorComputeProofFailed]	{"User":{}}
	computing seal proof failed(2): storage call error 0: Rust panic: no unwind information [Hostname: chaiwenyu-System-Product-Name]: Rust panic: no unwind information
12.	2022-08-24 22:24:37 +0800 CST:	[event;sealing.SectorRetryComputeProof]	{"User":{}}
13.	2022-08-24 22:28:46 +0800 CST:	[event;sealing.SectorComputeProofFailed]	{"User":{}}
	computing seal proof failed(2): storage call error 0: Rust panic: no unwind information [Hostname: chaiwenyu-System-Product-Name]: Rust panic: no unwind information
14.	2022-08-24 22:29:46 +0800 CST:	[event;sealing.SectorSealPreCommit1Failed]	{"User":{}}
	consecutive compute fails
15.	2022-08-24 22:30:46 +0800 CST:	[event;sealing.SectorRetrySealPreCommit1]	{"User":{}}

### Logging Information

```text
"error": "failed to compute proof: Rust panic: no unwind information"
```

**Comment by Reiers** [comment]
Hi @ChaiWenyu1983 

Thanks for the report. The issue you are experiencing lies elsewhere. (First glance, looks like a hardware issue)

Moving this issue to Lotus Discussions for help and troubleshooting.

Closing Lotus issue ticket.

**Comment by Reiers** [comment, marked as answer]
https://github.com/filecoin-project/lotus/discussions/9209#discussioncomment-3468003

Answered here!
